R/mrmash_wrapper.R

```diff
@@ -92,12 +92,15 @@ mrmash_wrapper <- function(X,
                            w0_threshold = 1e-8,
```

Comment:
This threshold is still here. Is it ever used? If not, please make sure subsequent steps uses thsi parameter.

Reply:
That's the mr.mash parameter to drop components with weight smaller than that value within the algorithm. So, it should be left, I think?